code-challenges.test.js

```diff
@@ -23,27 +43,132 @@ const secretCodeWord3 = "Eccentric"
 
 // b) Create the function that makes the test pass.
 
+// 1) Create a function that takes in a string and returns a string with a coded message. The coded message converts "a" to 4, "e" to 3, "i" to 1, and "o" to 0.
+
 // Pseudo code:
+// input: a string "Lackadaisical"
+// output: a string with the numbers converted to numbers returning a mix of numbs and letters 
+//         "L4ck4d41s1c4l", "G0bbl3dyg00k", "3cc3ntr1c"
+// process: turn string to array, .split()
+//          iteration is key here, need to access each value
+//          how to know if/what letter is what
+//          set a,e,i,o to absolutely equal the numbers 4,3,1,0
+//          for loop or .map() since i am trading out letters for numbers
+//          maybe conditional? Do conditionals solve all problems lol
+//          turn arr back to strign .join()
```

Comment:
This is some good pseudo code Megan. You're given yourself a great plan to work off of! Remember pseudo code are step not suggestions.